javaws/WebContent/index.html

```diff
@@ -45,14 +45,14 @@ <h3>Connection details</h3>
 </pre>
 If the output is something like
 <pre>
-Trying 188.184.36.76...
+Trying 10.116.96.88...
```

Comment:
Shouldn't that be 10.116.96.89, also below?

Reply:
urgh, yes, thanks for the catch :)